### PR TITLE
Add simple webchat.

### PR DESCRIPTION
**Pull request by ivan-karlovich**

### CODE REVIEW

File: server/request_handler.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  def do_GET(self):
+    print(f"GET method is called: {self.path}")
+
+    path = self.path
+    if (path == "/"):
+      path = "/index.html"
+
+    last_slash_index = path.rfind("/")
+    resource_name = path if last_slash_index == -1 else path[(last_slash_index + 1):];
```
Comment (RomanPlusPlus) [inline server/request_handler.py]: a semicolon at the line's end

File: server_bot.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "-p",
+    "--port",
+    type = int,
+    default = DEFAULT_PORT,
+    help = "Specify the port on which the server listens",
+  )
+
+  args = parser.parse_args()
+
+  return args;
```
Comment (RomanPlusPlus) [inline server_bot.py]: a semicolon at the line's end

**Review by RomanPlusPlus — APPROVED** [review]
Looks good! Only a minor nit: trailing semicolons in a few lines

File: server/request_handler.py (hunk 1 above)
Comment (ivan-karlovich) [reply server/request_handler.py]: Fixed.

File: server_bot.py (hunk 2 above)
Comment (ivan-karlovich) [reply server_bot.py]: Fixed.

**Pull request merged**